timemachine/cpp/generate_stubs

```diff
@@ -8,3 +8,4 @@ stubgen --output . --module "$MODULE"
 # Workaround for
 # > error: "ndarray" expects 2 type arguments, but 1 given
 sed -i 's/numpy\.ndarray\[\([^]]*\)\]/numpy.typing.NDArray[\1]/g' "${MODULE}.pyi"
+sed -i 's/from typing import List/from typing import List, Set/g' "${MODULE}.pyi"
```

Comment:
Nice workaround! Unfortunate that this seems necessary.

Reply:
I have no idea what's going on here

Reply:
I think this is to deal with the fact that mypy `stubgen` doesn't seem to add the required imports to `.pyi` files. (Might be unnecessary if we change the type of `col_atom_idxs` from `set` to `vector`?)

Reply:
Right. This was needed due to a missing import, removed since we are using vector now.